### DISCUSSION TITLE
Bluetooth: ImportError: no module named 'bluetooth'

### DISCUSSION THREAD

**Discussion by ripkk** [micropython/micropython#13211] [RP2040 / Pico] [answered]
Hi guys,

As the title suggests I'm getting the following error while trying to use bluetooth.

```
MicroPython v1.21.0 on 2023-10-06; Raspberry Pi Pico with RP2040
```

```
from machine import Pin,SPI
import framebuf
import time
from micropython import const
import bluetooth
```

I'm using Thonny as my IDE.

I've also tried the latest nightly builds which results in the same error.

Thanks for the help.

**Comment by mendenm** [comment, marked as answer]
It looks like you installed the Pico firmware, and not the Pico-W firmware with support for the WiFi stuff.

**Reply by ripkk** [reply]
Oh gosh, you are correct. 

Thanks for pointing that out :)
